diff -ur piratewallet-light-cli/lib/Cargo.toml piratewallet-light-cli-gentoo/lib/Cargo.toml
--- piratewallet-light-cli/lib/Cargo.toml	2021-03-15 11:42:45.000000000 +0900
+++ piratewallet-light-cli-gentoo/lib/Cargo.toml	2021-04-20 21:19:30.459245588 +0900
@@ -46,34 +46,28 @@
 reqwest = { version = "0.10.8", features = ["blocking", "json"] }
 
 [dependencies.bellman]
-git = "https://github.com/zerocurrencycoin/librustzcash.git"
-rev = "2981c4d2860f7cd73282fed885daac0323ff0280"
+version = "0.6.0"
 default-features = false
 features = ["groth16", "multicore"]
 
 [dependencies.pairing]
-git = "https://github.com/zerocurrencycoin/librustzcash.git"
-rev = "2981c4d2860f7cd73282fed885daac0323ff0280"
+version = "0.16.0"
 
 [dependencies.zcash_client_backend]
-git = "https://github.com/zerocurrencycoin/librustzcash.git"
-rev = "2981c4d2860f7cd73282fed885daac0323ff0280"
+version = "0.2.0"
 default-features = false
 
 [dependencies.zcash_primitives]
-git = "https://github.com/zerocurrencycoin/librustzcash.git"
-rev = "2981c4d2860f7cd73282fed885daac0323ff0280"
+version = "0.2.0"
 default-features = false
 features = ["transparent-inputs"]
 
 [dependencies.zcash_proofs]
-git = "https://github.com/zerocurrencycoin/librustzcash.git"
-rev = "2981c4d2860f7cd73282fed885daac0323ff0280"
+version = "0.2.0"
 default-features = false
 
 [dependencies.ff]
-git = "https://github.com/zerocurrencycoin/librustzcash.git"
-rev = "2981c4d2860f7cd73282fed885daac0323ff0280"
+version = "0.6.0"
 features = ["ff_derive"]
 
 [build-dependencies]
