diff -ur a/lib/Cargo.toml b/lib/Cargo.toml
--- a/lib/Cargo.toml	2021-01-26 09:02:27.000000000 +0900
+++ b/lib/Cargo.toml	2021-04-19 10:35:20.822536000 +0900
@@ -12,7 +12,7 @@
 rand = "0.5"
 hex = "0.3"
 bech32 = "0.6"
-zcash_primitives = {  git = "https://github.com/zcash/librustzcash", rev = "5a48d179b8434a0318e7e19dda506e245a904092" }
+zcash_primitives = "0.0.0"
 json = "0.11.14"    
 qrcode = { version = "0.8", default-features = false }
 printpdf = { version = "0.2.8", optional = true }
